package.json

```diff
@@ -8,8 +8,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "./node_modules/rimraf/bin.js ./build && npx tsc",
-    "start": "npm run build && node build/server.js"
+    "build": "rimraf build && tsc",
```

Comment:
Using rimraf globally didn't seem to work on Heroku. I changed it in master to use npx.

Reply:
Are you sure? It worked in my deploy: npm scripts should all run in the same context as npx (that's why you never see npx used in package.json files.

Reply:
When I first deployed to Heroku my build crashed until I put this cumbersome './node_modules/rimraf/bin.js ./build' thing. It defaulted to Node 12 initially, which I later fixed by specifying Node 14.1. Perhaps their Node 14 environment is more well-behaved in that respect.

Reply:
I'm not sure why it would have had an issue, the whole point of the scripts is that they run in the npm context. Not a big deal though.

I'm also wondering why you need node 14. The MRe package only requires 8, and I'm not seeing anything else that would need the bleeding edge node features.